common/test/acceptance/tests/lms/test_oauth2.py

```diff
@@ -56,5 +56,6 @@ def test_accepting_redirects(self):
         self.oauth_page.confirm()
 
         # This redirects to an invalid URI.
-        query = self._qs(self.browser.current_url)
+        self.oauth_page.wait_for_element_absence('input[name=authorize]', 'Authorization button is not present')
+        query = self._qs(self.browser.current_url) or self._qs(self.browser.title)
```

Comment:
Please split this out into an if/else block that is based on browser.name. That will make the code easier for future developers to understand and maintain.
Something like:
```
        if self.browser.name is 'chrome':
            query = self._qs(self.browser.title)
        else:
            query = self._qs(self.browser.current_url)
```